lib/index.js

```diff
@@ -58,8 +58,10 @@ api.db = null;
 // shared collections cache
 api.collections = {};
 
+const {writeConcern} = bedrock.config.mongodb.writeOptions;
+
 // default database write options
-api.writeOptions = bedrock.config.mongodb.writeOptions;
+api.writeOptions = {...bedrock.config.mongodb.writeOptions};
```

Comment:
We should mark this as deprecated and remove it in the next major release.

Reply:
marked with a note here: https://github.com/digitalbazaar/bedrock-mongodb/pull/65/commits/163a951857899ac70ee2944a892eb1b6eda2a484